bundles/org.openhab.io.neeo/README.md

```diff
@@ -193,16 +192,16 @@ If the device does not have power state or doesn't support input switching, the
 
 This following device capabilities are available:
 
-1.  "Always On" - check if there is no power management for the device.
+1. "Always On" - check if there is no power management for the device.
 
 You do NOT need to specify any POWER buttons or POWER STATE sensor nor will the device be marked as 'stupid'.
 
 #### Example
 
 In the example screen shown above:
 
-1.  The Russound AM/FM tuner was mapped to an "ACCESSORIE" NEEO type.
-2.  The Russound Great Room zone was mapped to an "AUDIO" device (to allow volume keys to work).
+1. The Russound AM/FM tuner was mapped to an "ACCESSORIE" NEEO type.
+2. The Russound Great Room zone was mapped to an "AUDIO" device (to allow volume keys to work).
```

Comment:
This is not a request for changing anything, just want to mention that it can be written like this, which will ensure automatic renumbering if anything is added in between:
```suggestion
1. The Russound AM/FM tuner was mapped to an "ACCESSORIE" NEEO type.
1. The Russound Great Room zone was mapped to an "AUDIO" device (to allow volume keys to work).
```

Reply:
Yes i know, might change this in the future.